.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.4
```

Comment:
Set development Ruby version to 2.4.4 which is [the oldest current stable Ruby release](https://www.ruby-lang.org/en/downloads/).